server/build/webpack.js

```diff
@@ -306,9 +301,34 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
           return false
         }
       }),
-      !isServer && new webpack.optimize.CommonsChunkPlugin({
+      dev && !isServer && new webpack.optimize.CommonsChunkPlugin({
         name: 'manifest',
         filename: 'manifest.js'
+      }),
+      !dev && !isServer && new webpack.optimize.CommonsChunkPlugin({
+        name: 'main.js',
+        filename: 'app.js',
```

Comment:
We can keep this as `main.js`.
And in the `server/document.js` we can change to use `main.js` instead of `app.js` for both production and dev.

For dev you may need to use `manifest.js` before `main.js`.

Reply:
Done